internal/nodes/avi_model_routeingr_hostname_shard.go

```diff
@@ -529,3 +540,18 @@ func updateHostPathCacheV2(ns, ingress string, oldHostMap, newHostMap map[string
 		}
 	}
 }
+
+func buildL7RouteInfraSetting(key string, vs *AviVsNode, vsvip *AviVSVIPNode, routeIgrObj RouteIngressModel) {
```

Comment:
Nit: I think this function can be in the file avi_model_l7_translator.go with buildL7IngressInfraSetting